src/main/java/frc/robot/subsystems/leds/LEDIOSim.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.leds;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.util.Color;
+
+// This is a copy of LEDIOReal right now, but if we move leds to separate ports sim breaks w
+// LEDIOReal
+public class LEDIOSim implements LEDIO {
+  private final AddressableLED led;
+  private final AddressableLEDBuffer buffer;
+
+  public LEDIOSim() {
+    led = new AddressableLED(3);
+    buffer = new AddressableLEDBuffer(LEDSubsystem.LED_LENGTH);
+    led.setLength(buffer.getLength());
```

Comment:
group similar ideas together for consistency with rest of codebase.

```suggestion
    buffer = new AddressableLEDBuffer(LEDSubsystem.LED_LENGTH);
    
    led.setLength(buffer.getLength());
```